src/RouteJs/RouteJs.cs

```diff
@@ -35,6 +42,21 @@ IEnumerable<IConstraintsProcessor> constraintsProcessors
 		}
 
 		/// <summary>
+		/// Gets or sets whether urls are converted to lower case.
+		/// </summary>
+		public static bool LowerCaseUrls { get; set; }
```

Comment:
Looks like `LowerCaseUrls` was added in .net 4.5. Your project targets .net 4.0. Thoughts?